### ISSUE TITLE
FR: Allow to disable printing "--help Show help message" for each subcommand

### ISSUE THREAD

**Issue by akozhemiakin** [scallop/scallop#128]
It would be awesome to be able to disable printing "--help Show help message" for each subcommand. One can decide that it is enough to inform user about possibility to call "--help" on any subcommand once, somewhere on the top level of the help message, and it is not necessary to explicitly specify it for each subcommand.

For instance I have this as a part of my help message and it looks pretty noisy. 
```
Subcommand: product db-migrate - Flyway migrate
      --help   Show help message
Subcommand: product db-baseline - Flyway baseline
      --help   Show help message
Subcommand: product db-clean - Flyway clean
      --help   Show help message
Subcommand: product db-validate - Flyway validate
      --help   Show help message
Subcommand: product db-repair - Flyway repair
      --help   Show help message
```

**Comment by Rogach** [comment]
Yes, currently there is no way to do it. You can override help option with your own and use hidden option:

```scala
object Conf extends ScallopConf(Seq()) {
  val sub = new Subcommand("sub") {
    val help = opt[Boolean](noshort = true, hidden = true)
  }
  addSubcommand(sub)
  verify()
}
Conf.printHelp()
```

But that only removes "Show help message" option description, the "--help" is still printed:
```
      --help   Show help message

Subcommand: sub
  -h, --help
```
 
I will fix this behavior. Will such workaround work for you?

**Comment by Rogach** [comment]
Published `2.0.7` with this change.
